_data/authors.yml

```diff
@@ -0,0 +1,26 @@
+rohitrox:
+  username: rohitrox
+  name: Rohit Joshi
+  location: Kathmandu, Nepal
+  twitter-username: "rohitroxxxxx"
```

Comment:
roxxypoxxy

Reply:
What does roxxypoxxy mean ?

Reply:
It's @RohitRox's twitter username https://twitter.com/roxxypoxxy

Reply:
That means I have to change the bio and image link too, And I changed It :D